code100x/cms-mobile
# Add "Project Page" section to Account Page

#### Issue Reference:
This PR is related to issue #12  - "Add 'Project Page' section to Account Page"

#### Description:
This PR introduces a new section called **Project Page** in the Account Page of the app. This section allows users to manage and view their course-related projects.

#### Changes Made:
- Added a new section to the Account Page called "Project Page".
- Added a button for adding new project.
- The section is responsive and follows the app's design language.

#### How to Test:
1. Navigate to the Account Page.
2. Verify that the new "Project Page" section is visible.
3. Ensure responsiveness on both iOS and Android devices.

#### Screenshots:

https://github.com/user-attachments/assets/d3752758-34bb-41f6-ad1a-141a6cd923dd



#### Additional Context:
This feature is a UI-only implementation for now, and backend functionality will be added in future PRs.
